federation/pkg/kubefed/init/init.go

```diff
@@ -463,9 +463,9 @@ func createAPIServer(clientset *client.Clientset, namespace, name, image, creden
 						},
 						{
 							Name:  "etcd",
-							Image: "quay.io/coreos/etcd:v2.3.3",
+							Image: "gcr.io/google_containers/etcd:3.0.14-alpha.1",
 							Command: []string{
-								"/etcd",
+								"/usr/local/bin/etcd",
```

Comment:
updated the PR, just to solve the failing test cases for now. we need to migrate to etcd3 asap.